frontend/src/metabase/public/components/widgets/SharingPane/icons/StaticEmbedIcon/StaticEmbedIcon.tsx

```diff
@@ -0,0 +1,45 @@
+import { StaticEmbedIconRoot } from "metabase/public/components/widgets/SharingPane/icons/StaticEmbedIcon/StaticEmbedIcon.styled";
+
+export const StaticEmbedIcon = ({ className }: { className?: string }) => (
```

Comment:
You can remove `className` here (and line 5), since we're now styling `StaticEmbedIconRoot` directly.